### PR TITLE
Adding a remote access guide

### PR DESCRIPTION
**Pull request by mckimb**
Ready for @dennissergeev and @rosscastle to review my edits. This addresses #160 or #161 or #162.

### CODE REVIEW

File: docs/source/remote_access.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
+Remote Access
+==============
+
+Overview
+-------
```
Comment (dennissergeev) [inline docs/source/remote_access.rst:5]: fix the length of the underline (extra `-`)

File: docs/source/remote_access.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
+Remote Access
+==============
```
Comment (dennissergeev) [inline docs/source/remote_access.rst:2]: remove one `=`

File: docs/source/remote_access.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+This command will generate 2 files:the one with the ``.pub`` extension is the "public key", the one without the ``.pub`` extension is the "private key". You keep your private key strictly on your local machine. You need to copy your public key to the remote machine you would like to use the key pair to log in to. First create the ``~/.ssh`` on your **remote machine**. Then from your **local machine** enter: ``$ scp ~/.ssh/id_rsa_hostalias.pub hostalias:~/.ssh/``. After copying the public key over to ``hostalias``, you need to create an ``authorized_keys`` file in your ``~/.ssh/`` on your **remote machine**.
```
Comment (dennissergeev) [inline docs/source/remote_access.rst:35]: Add a note that it's also good to change permissions of the private key file

File: docs/source/remote_access.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+Edit remote files in a local text editor using ``rmate``
+--------------------------------------------------------
+As an alternative to remote-based text editors such as ``vi`` and ``emacs``, we can
+use port-fowarding to set up a `local-based text editor like Atom <https://atom.io>`_ which includes features such as syntax highlighting and code completion. For instructions to install ``rmate`` on your **local machine**, `click here <https://github.com/textmate/rmate>`_. Then to specifically use Atom to edit remote files, `click here <https://atom.io/packages/remote-atom>`_. You will need to add the following line to your ``~/.ssh/config`` file: ``RemoteForward 52698 localhost:52698``.
+
+
+Edit remote python files in a ``jupyter`` environment
+-----------------------------------------------------
+The ``jupyter`` environment is a great environent for data exploration and integrating
```
Comment (dennissergeev) [inline docs/source/remote_access.rst:57]: Could also mention that you can send the command by ssh w/o logging to the remote host:
`ssh remotehost "jupyter lab --no-browser --port=3039"`

File: docs/source/remote_access.rst [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+   Hostname **hostname**
+      User **username**
+      LocalForward 3039 localhost:3039
+      RemoteForward 52698 localhost:52698
+      IdentityFile **~/.ssh/id_rsa_hostname**
+      UseKeychain yes
+      AddKeysToAgent Yes
+
+**One final note:** Remember the port numbers chosen are arbitary. If you choose the same number as someone else on your network, their files may open up on your computer and vice versa!
```
Comment (dennissergeev) [inline docs/source/remote_access.rst:76]: Another note: do not use this with VPN, use ithome aka hashbang as proxy

File: docs/source/remote_access.rst [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+Authors
+-------
+This documentation was written by Brett McKim, peer reviewed by Dennis Sergeev, and quality controlled by Ross Castle.
```
Comment (dennissergeev) [inline docs/source/remote_access.rst:80]: Denis (one "n" confusingly enough!)

File: docs/source/remote_access.rst (hunk 5 above)
Comment (dennissergeev) [inline docs/source/remote_access.rst:76]: Also add a note that if connection is interrupted you can still reconnect, assuming the jupyter process is still running. But make sure not to leave zombie jupyter processes with open ports on remote hosts!

**Comment by dennissergeev** [comment]
Just a few minor edits required, otherwise well done @mckimb!

**Review by rosscastle — APPROVED** [review]

**Review by dennissergeev — APPROVED** [review]
LGTM

**Pull request merged**